src/redux/reducers/roomReducer.js

```diff
@@ -0,0 +1,27 @@
+import * as types from '../actions/actionTypes';
```

Comment:
I'm confused. How much of this was taken from my branch?

In any case, you can leave it as-is, and I can worry about conflicts. If you like. :)